port scanner.py

```diff
@@ -0,0 +1,26 @@
+import socket
```

Comment:
Rename module to pythonic-form and add `sync` before it so indicate that this is the synchronous version of port scanner.